### PR TITLE
feat(tagger): topic links checker

### PR DESCRIPTION
**Pull request by kptdobe**
Linkchecker goodie in the tagger: this will check if the topic page corresponding to each topic (either link computed based on topic name or link defined on the topic itself in the taxonomy document) exists or not and provide a report. To test: http://topic-linkchecker--theblog--adobe.hlx.page/tools/tagger/index.html?checkLinks=true

### CODE REVIEW

File: tools/tagger/index.html [hunk 1]
```diff
@@ -400,11 +511,89 @@
...
+              const icon = document.createElement('span');
+              icon.classList = 'lcIcon';
+              icon.innerHTML = LOADING_ICON;
+              icon.title = `Checking if ${link} exists.`
+              ui.append(icon);
+              icon.addEventListener('click', () => {
+                window.open(link);
+              });
+
+              const res = await fetch(link);
```
Comment (rofe) [inline tools/tagger/index.html:572]: Would using a `HEAD` request make it a bit faster?

**Review by rofe — APPROVED** [review]

File: tools/tagger/index.html (hunk 1 above)
Comment (kptdobe) [reply tools/tagger/index.html:572]: Good idea!

Comment (kptdobe) [reply tools/tagger/index.html]: Interestingly, the `HEAD` requests seem to be much slower. I assume the `GET` might be cached somewhere from a previous hit while the `HEAD` request will always be new.

Comment (kptdobe) [reply tools/tagger/index.html:587]: Reverted to `GET` requests since definitively faster.

**Pull request merged**